manuscript/draft.Rmd

```diff
@@ -1,47 +1,58 @@
-# Title
+# Moving towards large-scale data-intensive forecasting of species richness in breeding birds
```

Comment:
Thoughts on:

*Using best practices for forecasting biodiversity changes in breeding birds*

as a middle ground?